src/AddressSplitter.php

```diff
@@ -1,40 +1,52 @@
 <?php
-namespace VIISON\AddressSplitter;
+namespace Pickware\AddressSplitter;
 
-use VIISON\AddressSplitter\Exceptions\SplittingException;
+use RuntimeException;
 
 /**
- * @copyright Copyright (c) 2017 VIISON GmbH
+ * @copyright Copyright (c) 2019 Pickware GmbH
```

Comment:
- Do we really need the copyright in every single file?
- If we do, I'd prefer using a copyright header which also specifies the license instead of the phpdoc @copyright attribute
- We should probably omit the year number since it doesn't add an value and looks obsolete pretty quickly 😉

Reply:
We decided to remove the copyright headers from each file.